deposit/src/main/java/edu/unc/lib/deposit/transfer/TransferBinariesToStorageJob.java

```diff
@@ -187,13 +179,38 @@ private void transferDepositManifests(PID objPid, Resource resc, BinaryTransferS
             }
 
             PID manifestPid = getDepositManifestPid(objPid, manifestFile.getName());
+            transferFile(manifestPid, manifestUri, transferSession, resc, CdrDeposit.storageUri);
+        }
+    }
+
+    private void transferFile(PID binPid, URI stagingUri, BinaryTransferSession transferSession,
+            Resource resc, Property storageProperty) {
 
-            if (!isObjectCompleted(manifestPid)) {
```

Comment:
Do the isObjectCompleted and markObjectCompleted methods in AbstractDepositJob need to be removed?

Reply:
No, they are still being used by other jobs. It is reasonable to use objectCompleted in cases where the jena model isn't being updated, since those can get lost in cases where the jvm shuts down or the jena transactions are interrupted at the wrong time.